api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralQueries.java

```diff
@@ -8,6 +8,26 @@
 @Component
 public class ResumoTrimestralQueries {
 
+  public static String getPatientsWhoReceivedOneViralLoadResult(
+      int encounterType, int viralLoad, int viralLoadQualitative) {
+    String query =
+        "SELECT p.patient_id FROM "
+            + "patient p JOIN encounter e "
+            + "ON p.patient_id = e.patient_id JOIN obs o "
+            + "ON o.encounter_id = e.encounter_id "
+            + "WHERE p.voided = 0 AND e.voided = 0 AND o.voided = 0 AND "
+            + "e.encounter_type = ${encounterType} AND (o.concept_id = ${viralLoad} AND o.value_numeric IS NOT NULL) "
```

Comment:
@simaomazive the reason we using this stringsubstitutor is to be able to ready the query without having to go and see what encounterType he is talking about...
Please change this name to be more suggestive like 
```suggestion
            + "e.encounter_type = ${adultoSeguimentoEncounterType} AND (o.concept_id = ${viralLoadConcept} AND o.value_numeric IS NOT NULL) "
```

Reply:
See here https://github.com/jembi/openmrs-module-eptsreports/blob/develop/api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java#L916